src/nvim/testdir/test_regexp_latin.vim

```diff
@@ -0,0 +1,18 @@
+" Tests for regexp in latin1 encoding
+set encoding=latin1
+scriptencoding latin1
+
+func Test_nested_backrefs()
```

Comment:
The test should pass after removing the `encoding=latin1` lines. Even though it's not testing the original intention, we want to keep it to make merges easier in the future.